Dockerfile

```diff
@@ -29,8 +29,10 @@ LABEL maintainer "Ansible <info@ansible.com>"
 
 ENV PACKAGES="\
     docker \
+    openssh-client \
     ruby \
     "
+
```

Comment:
FTR I intentionally didn't add this line, because both vars are packages installed with apk so I wanted to group them.